xcodeproj/internal/xcodeproj.bzl

```diff
@@ -99,7 +99,8 @@ def _write_json_spec(*, ctx, project_name, configuration, inputs, infos):
 "label":"{label}",\
 "name":"{name}",\
 "target_merges":{target_merges},\
-"targets":{targets}\
+"targets":{targets},\
+"scheme_autogeneration_mode":"{scheme_autogeneration_mode}"\
```

Comment:
Can this be moved up to be alphabetical?

Reply:
Yes. Apparently, I am pretty sleepy this afternoon. I thought that it was alphabetical. 😳